static/include/components/base-sensor.md

```diff
@@ -0,0 +1,13 @@
+If you want to control your base by specifying a desired velocity in terms of distance/time, for example 5 m/s and you have a sensor, for example a wheel encoder, that measures the actual velocity, you can make use of the sensor to adjust the velocity.
+
+One way to adjust velocity based on a sensor's measurements, is using a PID control loop that measures the velocity of movement and determines the difference or error from the desired velocity to compute a correction for the base based on three terms:
```

Comment:
```suggestion
The 'sensor-controlled' model of Base uses a PID control loop that measures the velocity of movement and determines the difference or error from the desired velocity to compute a correction for the base based on three terms:
```

Reply:
Updated suggestion after realizing that if no control_parameters are provided, it will default to using the basic "10% increments" form of control.

Reply:
not sure sensor controlled base is doing this 10% thing, I'll have to take a closer look at the code. rand wrote that, but I know RPM monitor (which is only used by encoded gpio motor) does the 10% thing

Reply:
I see, gotcha. Would you mind checking if sensor controlled base defaults back to 10%?

Reply:
okay so sensor controlled base without PID calls SetVelocity on the underlying base (specified already in the config). wheeled base just does some math and then calls go for on the underlying motors. so if these motors _also_ don't have PID values configured it do the +- 10% increment. if those underlying motors do have PID values configured it will use that PID loop for better performance. but I don't think we should directly say how the base will behave without PIDs configured because we don't know what the users underlying base is/does. maybe it's enough to say it will just call `SetVelocity` on the dependent base?

Reply:
oof. that is definitely complicated!

so does wheeled base do the +- 10%? if so, agree we can say if control_parameters are not set, sensor-controlled base delegates to the underlying base

Reply:
sometimes. wheeled base doesn't really know what it's doing, it just calls go for on it's motors. if those motors don't have the control parameters configured then it defaults to +/- 10% power

Reply:
and we don't have a way of knowing from wheeled base if the underlying motors have PID values configured

Reply:
ik this is confusing so if you want to discuss in person (or if you do @npentrel) feel free to put some time on my calendar!

Reply:
got it! it makes sense now. updated the suggested edit one more time.

Reply:
i think one of the intermediate versions of this got applied - resolving this comment and adding a new one. sorry about the confusion